src/build/get_build_environment.mjs

```diff
@@ -40,9 +44,11 @@ export async function getBuildEnvironment(platform, buildMode, sentryDsn) {
     }
   }
 
+  const [, appVersion] = candidateId?.match(/\/?v?([[0-9]+\.[0-9]+\.[0-9]+)/) ?? [];
+
   return {
     SENTRY_DSN: sentryDsn,
-    APP_VERSION: `${await getVersion(platform)}${buildMode === 'debug' ? '-debug' : ''}`,
-    APP_BUILD_NUMBER: await getBuildNumber(platform),
+    APP_VERSION: buildMode === 'release' ? appVersion : `${appVersion}-${buildMode}`,
```

Comment:
If candidateId is missing, I think appVersion is undefined, so this will be `-debug`?
Perhaps we just omit or make it `0.0.0-debug`?